app/controllers/enterprises_controller.rb

```diff
@@ -10,14 +10,14 @@ class EnterprisesController < BaseController
   before_filter :check_stock_levels, only: :shop
 
   before_filter :clean_permalink, only: :check_permalink
+  before_filter :set_enterprise, only: :relatives
```

Comment:
why don't you call the method from the controller action? I believe single-action `before_filter`s have little to no value. It is easier for the reader to see a method call when reading the action than jumping between the `before_filter` and the action.

Reply:
Resolved